packages/material-ui/test/__snapshots__/UpDownWidget.test.tsx.snap

```diff
@@ -18,14 +18,14 @@ exports[`renders 0 as 0 and not '' 1`] = `
     </span>
   </label>
   <div
-    className="MuiInputBase-root MuiInput-root MuiInput-underline Mui-disabled Mui-disabled MuiInputBase-formControl MuiInput-formControl"
+    className="MuiInputBase-root MuiOutlinedInput-root Mui-disabled Mui-disabled MuiInputBase-formControl"
```

Comment:
This indicates that this test was likely run with the material UI v5 code

Reply:
@epicfaace Do you automate snapshot updates? I wasn't sure if I should include them or not. I had to update the snapshots to get the tests to pass, locally.

Reply:
You need to update the snapshots yourself in this PR.